spec/repository_spec.rb

```diff
@@ -26,3 +26,17 @@
     expect(EXTRA_FILES).to eq([])
   end
 end
+
+describe "The file \"renamed-styles.json\"" do
+  it "must be valid JSON" do
+    expect(JSON_IS_VALID).to be true
+  end
```

Comment:
You could time the actual test to see if the test takes so long (i.e., time the expect line only)... otherwise it could also have something to do with the way we're building up the examples dynamically.